bezsmertnyi-stanislav/Readme.MD

```diff
@@ -0,0 +1,68 @@
+Flashscore Live Scores
+======================
+
+## Introduction
+
+The Flashscore Live Scores app provides real-time sports scores and updates from various sporting events worldwide. Stay up-to-date with the latest scores, fixtures, and results for your favorite sports and teams. The most up-to-date live scores, results, stats, news and other info from football, tennis, basketball and 30+ sports and more than 6000 competitions from all around the world. You can select your favourite matches, teams or players, and our notifications will let you know about every important moment of the match.
+
+![](https://i.imgur.com/NRPPEeA.jpg)
+
+## Installation
+
```

Comment:
Notes on the installation procedure:
* When describing a procedure -- that is, steps taken in a certain order, -- always use a numbered list.
* Try formatting each list item as a complete sentence that ends with a period. For details, see https://learn.microsoft.com/en-us/style-guide/scannable-content/lists
* Don't cram several steps into one (as you did in the second list item).
* The installation procedure is not very clearly described. I'm not even sure on which platform we're supposed to install it -- PC or mobile device?